### PR TITLE
Update README.md [BAR-23]

### PR DESCRIPTION
**Pull request by josephkott**

### CODE REVIEW

**Comment by sashaaero** [comment]
Picture is quite small, is that on purpose?

File: readme.md [hunk 1]
```diff
@@ -1,149 +1,180 @@
...
 
-Information about stages:
+This step only needs to be done once.
+To download reference dataset one should use `reference` command of the pipeline launcher.
+The command below downloads needed references to the directory specified by the flag `--ref-directory`.
+After that `--ref-directory` argument should now be used in all subsequent commands.
 
-**Germline workflow**
+```bash
+docker run --rm --privileged -it -v /media:/media -v /etc/localtime:/etc/localtime:ro \
```
Comment (alex-medvedev-msc) [inline readme.md:37]: We can remove --priveleged flag. It was added for using singularity inside docker

File: readme.md [hunk 2]
```diff
@@ -1,149 +1,180 @@
...
-5. Close Relatives: KING IBD search.
-6. IBD Search: Germline with merging closely located IBD segments together.
-7. Distant Relatives: ERSA with default params estimated on CEU founders.
-8. Merge: KING degree has priority over ersa degree for close relatives (degrees 1-3), otherwise, we take ERSA output.
+If phasing and imputation are required (mainly for the GERMLINE workflow) one should specify additional `--phase` and `--impute` flags to previous command to download additional reference datasets.
+Total amount of required disk space is about **50GB**.
 
-**Fast IBIS workflow**
+```bash
+docker run --rm --privileged -it -v /media:/media -v /etc/localtime:/etc/localtime:ro \
```
Comment (alex-medvedev-msc) [inline readme.md:45]: Also remove --privileged

File: readme.md [hunk 3]
```diff
@@ -1,149 +1,180 @@
...
-5. Distant Relatives: ERSA with default params estimated on CEU founders.
-6. Merge: KING degree has priority over ersa degree for close relatives (degrees 1-3), otherwise, we take ERSA output.
+There is another options to download all required reference data as a single file.
+This file is prepared by us and preloaded on our side in the cloud.
+It can be done by specifying additional flag `--use-bundle` to the `reference` command.
+This way is faster, since all the post-processing procedures have been already performed.
 
-![image](https://user-images.githubusercontent.com/19895289/123947586-35c90900-d9a9-11eb-976a-b98b1a25ba5d.png)
+```bash
+docker run --rm --privileged -it -v /media:/media -v /etc/localtime:/etc/localtime:ro \
```
Comment (alex-medvedev-msc) [inline readme.md:56]: Also remove --privileged

File: readme.md [hunk 4]
```diff
@@ -153,256 +184,178 @@ g1-b1-i1 g2-b3-i1     1           PO                0.4999          0.2467
...
-![image](https://user-images.githubusercontent.com/19895289/123947763-6a3cc500-d9a9-11eb-8cf9-8bd8d47e6e33.png)
+<p align="center">
+    <img src="./precision & recall.png" alt="drawing"/>
+</p>
 
-#### How to run simulation
+## Evaluation on the Hapmap Data
 
-Use command simulate. Options --input and --samples are not needed in this case.
+Use command `hapmap` for the downloading and preparation of the Hapmap CEU data.
```
Comment (alex-medvedev-msc) [inline readme.md:330]: hapmap does not download hapmap CEU data. It uses CEU genomes from 1000 Genomes dataset that should be already downloaded.

**Comment by josephkott** [comment]
@sashaaero, should be fixed, please check it on your side.

File: readme.md (hunk 4 above)
Comment (josephkott) [reply readme.md]: Hapmap was removed completely from GRAME.

**Review by alex-medvedev-msc — APPROVED** [review]

**Pull request merged**